### ISSUE TITLE
Lack of visual focus indicator on homepage and navbar

### ISSUE THREAD

**Issue by jchristgit** [python-discord/site#1390]
As outlined by #1363:

Some elements get properly focussed with the keyboard, but this is not visually communicated to the user. Namely, the discord join link in the top bar and the embedded youtubbe video lack any form of focus indication.

Users who use a keyboard need to see what is currently in focus. Each focussed element should have an easily viewable  way to distinguish it, such as a high- contrast outline or shadow.
